installer/k8s/test_restart_api.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Why this file called `test_restart_api.sh` but i dont see any tests in this file.

Reply:
this file is not used anywhere so it may be removed?